src/index.js

```diff
@@ -13,7 +13,6 @@ import '@playkit-js/playkit-js-hls';
 import '@playkit-js/playkit-js-dash';
 // Import analytics plugin
 import 'playkit-js-analytics';
-import '@playkit-js/playkit-js-kava';
```

Comment:
You meant to remove `import 'playkit-js-analytics';`?

Reply:
No, I changed the `playkit-js-analytics` to look for file which import the relevant analytics for kaltura-player type.
kava was inserted into this file `analytics.js` under ott and ovp folders